osmtm/templates/task.gpx.mako

```diff
@@ -12,7 +12,7 @@ timestamp = timestamp.isoformat()
   <time>${timestamp}</time>
 </metadata>
 <trk>
-  <name>Task for project ${project_id}</name>
+  <name>${_('Task for project ${project}; do not edit outside of this box!', mapping={'project': project_id}) | n}</name>
```

Comment:
In this case, I would separate the sentence into two.
'Do not edit outside of this box!' could be translated only once. This makes less work for the translators.
I may be too picky though.